api/graphite.go

```diff
@@ -1037,3 +1038,109 @@ func (s *Server) clusterTags(ctx context.Context, orgId int, filter string, from
 
 	return tags, nil
 }
+
+func (s *Server) graphiteAutoCompleteTags(ctx *middleware.Context, request models.GraphiteAutoCompleteTags) {
+	tags, err := s.clusterAutoCompleteTags(ctx.Req.Context(), ctx.OrgId, request.TagPrefix, request.Expr, request.From, request.Limit)
+	if err != nil {
+		response.Write(ctx, response.WrapError(err))
```

Comment:
This needs to use the same JSON response format as graphite `{"error":"<message>"}`

Reply:
https://github.com/grafana/metrictank/pull/779/commits/5c2cdf97d46cd749f986d8f870da3a94e8641caf